packages/web-app-files/src/views/AppBanner.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <portal to="app.app-banner">
+    <div class="app-banner hide-desktop" :hidden="shouldShow === false">
+      <oc-button
+        variation="brand"
+        appearance="raw"
+        class="app-banner-exit"
+        aria-label="Close"
+        @click="close"
+      >
+        <oc-icon name="close" size="small" />
+      </oc-button>
+      <div
+        class="app-banner-icon"
+        :style="{ 'background-image': `url('${appBannerSettings.icon}')` }"
+      ></div>
+      <div class="info-container">
+        <div>
+          <div class="app-title">{{ appBannerSettings.title }}</div>
+          <div class="app-publisher">{{ appBannerSettings.publisher }}</div>
+          <div class="app-additional-info">{{ appBannerSettings.price }}</div>
```

Comment:
What kind of value is price to have? If it's supposed to be something like "free" in the config, we probably want a way to translate that. Or have a fallback to free here if price is not set. That way we could use our regular translation system

Reply:
Do we really want to have it? I mean, different locations, currencies, maintenance ....

Reply:
But this still depends on the market you provide the application.  Are we only in € concurrency countries?

Reply:
Also, do we maybe charge different on iOS/Android ?

Reply:
I  wondering about the whole topic because since when can the admin set this, store is mostly run by us

Reply:
Already discussed with @grimmoc , the field should be named `additionalInfo` - if we need it at all, which I'm not sure about to be honest. Ultimately, @tbsbdr knows best what potential users might need here. IMO just leave it out 🤷